### PR TITLE
gvr-3dcursor: Bugfixes for single IoDevice

### PR DESCRIPTION
**Pull request by parthmehta209**
1. Avoiding setting position of cursor in GearS2Button and HandCursorButton
   when IoDevice is GearVR
2. Changed activeCursors to enabledCursors in LaserCursorButton

### CODE REVIEW

**Comment by parthmehta209** [comment]
@rahul27 minor bugfix to sample app, please review.

File: gvr-3dcursor/app/src/main/java/org/gearvrf/io/cursor/CursorScript.java [hunk 1]
```diff
@@ -67,7 +67,7 @@
     private static final float SCENE_HEIGHT = 3.0f;
     private static final float SPACE_OBJECT_MARGIN = 30.0f;
     private static final float BUTTON_XORIENTATION = 30.0f;
-    private static final float POINT_CUR_POS_RESET_FACTOR = 0.8f;
+    private static final float OBJECT_CUR_POS_RESET_FACTOR = 0.8f;
```
Comment (rahul27) [inline gvr-3dcursor/app/src/main/java/org/gearvrf/io/cursor/CursorScript.java]: Let's change this to OBJECT_CURSOR_RESET_FACTOR to maintain consistency.

File: gvr-3dcursor/app/src/main/java/org/gearvrf/io/cursor/CursorScript.java [hunk 2]
```diff
@@ -920,7 +931,10 @@ void handleClickReleased(CursorEvent event) {
             }
 
             targetCursor.setCursorTheme(crystalTheme);
-            targetCursor.setPosition(cursorPosition.x, cursorPosition.y, cursorPosition.z);
+            if(targetCursor.getIoDevice()!= null && !targetCursor.getIoDevice().getDeviceId()
+                    .equals(GEARVR_DEVICE_ID)) {
+                targetCursor.setPosition(cursorPosition.x, cursorPosition.y, cursorPosition.z);
```
Comment (rahul27) [inline gvr-3dcursor/app/src/main/java/org/gearvrf/io/cursor/CursorScript.java]: Cleaner to create a private function to set position. You could add a check for GEARVR_DEVICE_ID whenever that method is invoked.

File: gvr-3dcursor/app/src/main/java/org/gearvrf/io/cursor/CursorScript.java (hunk 1 above)
Comment (parthmehta209) [inline gvr-3dcursor/app/src/main/java/org/gearvrf/io/cursor/CursorScript.java]: Changed it.

File: gvr-3dcursor/app/src/main/java/org/gearvrf/io/cursor/CursorScript.java (hunk 2 above)
Comment (parthmehta209) [inline gvr-3dcursor/app/src/main/java/org/gearvrf/io/cursor/CursorScript.java]: Done.

**Comment by rahul27** [comment]
:+1: looks good.

**Comment by thomasflynn** [comment]
thanks guys!

**Pull request merged**